client/src/components/Typeahead/Typeahead.tsx

```diff
@@ -75,12 +75,13 @@ export class Typeahead extends React.Component<TypeaheadProps, TypeaheadState> {
 
   constructor(props: TypeaheadProps) {
     super(props);
+    const { query_value, min_length } = props;
 
     this.menu_id = _.uniqueId("typeahead-");
 
     this.state = {
-      input_value: props.query_value,
-      may_show_menu: false,
+      input_value: query_value,
+      may_show_menu: query_value.length >= min_length,
```

Comment:
What's the case where this is necessary? Seems to imply there's a case where the Typeahead is initialized with a query active and we want the menu to open, without focus on the input?